v1.0/openapi3/schemas/AuthenticationType.yaml

```diff
@@ -3,5 +3,8 @@ type: string
 enum:
   - OTP
   - QRCODE
```

Comment:
Is it possible to add `U2F` here? Because it seems to be the preferred authorization type for third party transactions

Reply:
No. See my comment here https://github.com/mojaloop/api-snippets/pull/9#discussion_r456857644

Reply:
Ah I see, sorry hadn't checked the comments there before, thanks for clarifying!